### ISSUE TITLE
Sample collection contents

### ISSUE THREAD

**Issue by sundalei** [mongodb-the-definitive-guide-3e/mongodb-the-definitive-guide-3e#3]
I'm reading the aggregation part. It will be much better if I have the sample collection contents of companies. Can somebody share that.

**Comment by iafra** [comment]
I agree with sundalei. companies collection is missing and it makes the understandig not easy.
